autoscheduler/frontend/src/utils/timeUtil.ts

```diff
@@ -9,11 +9,13 @@ export const LAST_HOUR = 22;
  * @param use24Hour if true, applies 24-hour format instead of the default 12-hour format
  */
 export function formatTime(h: number, m: number, use24Hour = false, padZeroes = false): string {
+  let amOrpm = 'AM';
+  if (h >= 13) {
```

Comment:
```suggestion
  if (h >= 12) {
```
right? Because 12 noon is 12 PM?